### ISSUE TITLE
Android Action Bar tooltip empty onHover (Android)

### ISSUE THREAD

**Issue by rauliyohmc** [wix/react-native-navigation#1560]
### Issue Description
Android Action Bar tooltip is empty when hovering on an icon.
I tried using `hint` but it doesn't work, is there any way to achieve that?
```javascript
        rightButtons: [
          {
            id: 'date-range',
            icon: 'date-range',
            disableIconTint: true,
            hint: 'calendar',
          }
        ],
        ...
```

### Steps to Reproduce / Code Snippets / Screenshots
**Expected**
![image](https://user-images.githubusercontent.com/4982414/28542189-9f65cba6-70bb-11e7-9309-5b94cd307b18.png)

**Current**
![image](https://user-images.githubusercontent.com/4982414/28542310-0a7e0584-70bc-11e7-94ba-689731a958be.png)

### Environment
* React Native Navigation version: v1.1.143
* React Native version: 0.46.1
* Platform(s) (iOS, Android, or both?): Android
* Device info (Simulator/Device? OS version? Debug/Release?): All

**Comment by varungupta85** [comment]
I think you should set the title to calendar instead of hint.

**Comment by guyca** [comment]
We should disable the longPress listener if `title` isn't defined. Duplicates #1007

**Comment by rauliyohmc** [comment]
Thanks guys!

**Comment by guyca** [comment]
This can probably be achieved by overriding the long press listener on the actual view. [Here](https://github.com/wix/react-native-navigation/blob/master/android/app/src/main/java/com/reactnativenavigation/views/TitleBarButton.java#L84).

**Comment by bingfj2008** [comment]
@guyca 
Thank you for your answer.

**Comment by guyca** [comment]
fixed in 354f60d1247ca11dd870313d57d1d2f336d80609
